src/rust/ensogl/src/display/render/passes/symbols.rs

```diff
@@ -31,16 +31,21 @@ impl SymbolsRenderPass {
 
 impl RenderPass for SymbolsRenderPass {
     fn outputs(&self) -> Vec<RenderPassOutput> {
-        vec![ RenderPassOutput::new("color",texture::Rgba,texture::item_type::u8)
-            , RenderPassOutput::new("id",texture::Rgba32ui,texture::item_type::u32)
+        let color_parameters = texture::Parameters::default();
+        let id_parameters = texture::Parameters{
+            min_filter : Context::NEAREST,
+            mag_filter : Context::NEAREST,
+            ..Default::default()
+        };
+        vec![ RenderPassOutput::new("color",texture::Rgba,texture::item_type::u8, color_parameters)
+            , RenderPassOutput::new("id",texture::Rgba32f,texture::item_type::f32, id_parameters)
```

Comment:
How is it working? Seriously, I need proper explanation here. Accoridng to the table https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/texImage2D , the `Rgba32f` format is not color renderable. It should not work, right?